packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1173,6 +1174,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         className={css(this._classNames.callout, calloutProps ? calloutProps.className : undefined)}
         target={this._comboBoxWrapper.current}
         onDismiss={this._onDismiss}
+        onMouseDown={this._onCalloutMouseDown}
```

Comment:
What part of this change is getting rid of the onBlur invoking/removing the effect of clicks to headers, dividers and scrollbars?

Reply:
by adding `ev.preventDefault();` to the onMouseDown that has the effect to literally nullify that click. It never happened. So focus is never removed from the input. 

With the Dropdown, we control focus via state, so we have control over when focus is lost. Combobox uses natural HTML input focus, so we have to stop the click at the point of clicking, not later on in the process.